src/libs/billing.js

```diff
@@ -3,12 +3,12 @@ import { Ajax } from 'src/libs/ajax'
 import { ensureBillingScope } from 'src/libs/auth'
 
 
-export const listProjectsWithAccounts = async () => {
-  const { Billing, GoogleBilling } = Ajax()
+export const listProjectsWithAccounts = async accounts => {
+  const { GoogleBilling } = Ajax()
 
   await ensureBillingScope()
 
-  const accountNames = _.map('accountName', await Billing.listAccounts()) // or fetch account names from FireCloud
+  const accountNames = _.map('accountName', accounts)
```

Comment:
This changes `listProjectsWithAccounts` so that it is passed the list of accounts rather than generating the list itself. This saves us a call to generate the list of accounts since we (will) need that information elsewhere in the Billing page.